Algorithmization/src/com/company/task_4/Nod.java

```diff
@@ -0,0 +1,7 @@
+package com.company.task_4;
+
+public class Nod {
```

Comment:
for future, name such classes as Utils - `MathUtils` here for exampe
Util class should have private constrcutor since they should not be instantiated